packages/next/build/webpack/config/blocks/base.ts

```diff
@@ -46,7 +46,9 @@ export const base = curry(function base(
   if (!config.module) {
     config.module = { rules: [] }
   }
-  config.module.strictExportPresence = true
+
+  // TODO: add codemod for "Should not import the named export" with JSON files
+  config.module.strictExportPresence = !isWebpack5
```

Comment:
We should probably have a future rule that re-enables this behavior.

Extra nice would be to only disable this when webpack 5 is automatically enabled, but leaving the field enabled if the user explicitly turned on webpack 5.